cfme/tests/containers/test_manageiq_ansible_users.py

```diff
@@ -0,0 +1,144 @@
+import pytest
+from cfme.containers.provider import ContainersProvider
+from cfme.configure.access_control import User
+from cfme.web_ui import Table
+from utils import testgen
+from utils.ansible import setup_ansible_script, run_ansible, \
+    fetch_miq_ansible_module, create_tmp_directory, remove_tmp_files
+from utils.appliance.implementations.ui import navigate_to
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.7"),
+    pytest.mark.usefixtures('setup_provider')]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='module')
+
+table_el = '//table[contains(@class, ' \
```

Comment:
The table that I test in this test is defined by:
<table class="table table-striped table-bordered table-hover table-selectable">
It appears in this specific test and it's the only table in this view. I fail to see how this can be an issue going forward. If, or when this table will change the test will surely need to be changed as well. I'd like to hear a better solution if you have any